resources/notes.py

```diff
@@ -9,13 +9,18 @@
 
 class Note(Resource):
     @pm_level_required
-    def post(self, id):
-        TicketModel.find(id)
+    def post(self, ticket_id):
+        TicketModel.find(ticket_id)
         return NotesModel.create(
             schema=NotesSchema,
             payload={
                 "text": request.json["text"],
-                "ticketid": id,
+                "ticketid": ticket_id,
                 "userid": get_jwt_identity(),
             },
         ).json()
+
+    @pm_level_required
+    def delete(self, ticket_id, note_id):
+        NotesModel.delete(note_id)
+        return {"message": "Note deleted"}
```

Comment:
This needs to be in a class. `class Note`
Let's replace `note_id` with `id` - It's a small style thing, but it's how the rest of the app is doing so lets stay consistent